distribution/solr-distro/assembly.xml

```diff
@@ -37,6 +37,10 @@
         </dependencySet>
     </dependencySets>
     <fileSets>
+        <fileSet>
+            <directory>src/main/resources/solrFiles</directory>
```

Comment:
❗️ Please add a reference to the Solr ticket and that this should be fixed in Solr 7.5 so we make sure to remove this when we upgrade.

https://issues.apache.org/jira/browse/SOLR-12538

Reply:
I'm glad there is a ticket. I agree we should link to it in the assembly file. @Phil We ever create a ticket that says "when we upgrade to version X.Y.Z, we need to go back and change this or that?"

Reply:
We have.  Sometimes we even create the upgrade ticket on the DDF side and reference that too.